hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3ABlocksize.java

```diff
@@ -18,9 +18,12 @@
 
 package org.apache.hadoop.fs.s3a;
 
+import java.io.IOException;
+
 import org.apache.hadoop.fs.FileStatus;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.s3a.impl.InternalConstants;
```

Comment:
I think I will have to remove these import. Looks like got imported during experiments.